src/hooks/screen/useRootChannelListHook.ts

```diff
@@ -39,6 +120,7 @@ const useRootChannelListHook = () => {
   }, [signedChannelUnreadCount, anonymousChannelUnreadCount]);
 
   return {
+    checkNotificationPermission,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `checkNotificationPermission` function is being returned from the hook but it's not clear if it's being used elsewhere. If it's not, consider removing it to reduce complexity and improve maintainability.
```diff
- checkNotificationPermission,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->